src/main/kotlin/support/security/Encrypts.kt

```diff
@@ -0,0 +1,10 @@
+package support.security
+
+import java.security.MessageDigest
+
+private val SHA256 = MessageDigest.getInstance("SHA-256")
```

Comment:
플랫폼 자료형(`MessageDigest!`)이므로 명시적으로 자료형을 지정하면 어떨까요?

```suggestion
private val SHA256: MessageDigest = MessageDigest.getInstance("SHA-256")
```